### ISSUE TITLE
device side pdo mapping workaround 

### ISSUE THREAD

**Issue by xdaco** [KITmedical/kacanopen#14]
Hi I have added device side pdo mapping i.e pdo mapping in slave. The workaround can be seen in "tools"
https://github.com/xdaco/kacanopen-fork/tree/master/tools

**Comment by roboticlemon** [comment]
Excellent addition xdaco, would you consider expanding permissions on your repo to accept PRs? This repo seems to have lost it's maintainers.

**Comment by xdaco** [comment]
@ingramator are you working on this actively ? If so, then lest discuss over email. I have some plans. We can together figure out a strategy to maintain this excellent work. I have been working a lot on this to our internal need.

**Comment by roboticlemon** [comment]
Yep I've been using it for a few months but just started working on it! It's probably the most user friendly piece of canopen c++ client code I've seen so it's definitely worth maintaining. 

I've sent an email to your git contributor email.
